### ISSUE TITLE
Using compact forms with Link fields

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/compact_forms#14]
It seems like link fields may not work with compact forms.

> ![Screen Shot 2020-09-02 at 10 24 37 PM](https://user-images.githubusercontent.com/871421/92067922-1b170b00-ed6b-11ea-91a1-f22eee6197fb.jpg)

**Comment by ghost** [comment]
Yep, they're tricky. They have `#type` = `link_field`, but because they're made up of two seperate fields (Title and URL) then just adding the `link_field` type to the code doesn't work... Need a different solution for these.
